dapr/proto/runtime/v1/dapr.proto

```diff
@@ -682,7 +685,7 @@ message WorkflowReference {
 
 message GetWorkflowRequest {
   string instance_id  = 1;
-  string workflow_type = 2;
+  string workflow_name = 2;
```

Comment:
This will be a breaking change in the generated SDKs

Reply:
Yes.